src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/hash-map/hash-map)
**Using public access modifiers everywhere**
If the method has only a utility purpose and is used only inside the same class, it should not be public. **Keep your code as close as possible** to follow the encapsulation principle.

**Remember about access modifiers.**
All fields and methods should have access modifiers. Remember about the encapsulation principle.